Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -102,23 +188,42 @@ private IPackageManager CreatePackageManager(string rootDir)
             return mgr;
         }
 
-        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath)
+        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath, params IPackage[] dependencies)
```

Comment:
My understanding is that RoslynGenTests is an integration test file and AnalyzerPluginGeneratorTests are unit tests, so they are kept seperate.